oak/server/logging_node.cc

```diff
@@ -47,8 +50,12 @@ void LoggingNode::Run(Handle handle) {
       if (result.msg == nullptr) {
         break;
       }
+      oak::logging::LogMessage log_msg;
+      std::string msg_str(result.msg->data.begin(), result.msg->data.end());
```

Comment:
It's slightly annoying to have to make this extra copy, is there any way to parse the protobuf message directly from the bytes field? Failing that, perhaps creating a `string_view` would at least avoid the copy, even though it would still require an explicit conversion here?

Reply:
I could not make it work with `string_view`. Apparently the compiler does not recognize it! 
This is the code I used: 
```  
      oak::log::LogMessage log_msg;
      std::string_view str(result.msg->data.data(), result.msg->data.size());
      log_msg.ParseFromString(str.data());
```
And the error message is: `no member named 'string_view' in namespace 'std'`.

I also tried to parse the protobuf directly: `log_msg.ParseFromString(result.msg->data.data());`, but it results in memory corruption, when running the examples with `-s asan`. We need a null-terminated string here, and for that we have to make a copy. Can you think of a work around?

Reply:
I think `string_view` is in `absl`, not in `std` yet (but i have not checked).

Anyways, it seems maybe https://developers.google.com/protocol-buffers/docs/reference/cpp/google.protobuf.message_lite#MessageLite.ParseFromArray would do the trick here?

Reply:
Fixed using `ParseFromArray`. Thanks for suggesting it @tiziano88.